### PR TITLE
Update changelog for 0.53.0rc3

### PR DESCRIPTION
**Pull request by sklam**
as titled

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -135,6 +135,7 @@ Fixes:
 * PR #6741: Enable SSA in IR inliner (Ehsan Totoni)
 * PR #6763: use an alternative constraint for the conda packages (Valentin
   Haenel)
+* PR #6786 Fix gufunc kwargs support (Siu Kwan Lam)
```
Comment (stuartarchibald) [inline CHANGE_LOG:138]:
```suggestion
* PR #6786: Fix gufunc kwargs support (Siu Kwan Lam)
```

**Comment by esc** [comment]
Will this need to be cherry-picked onto the release branch for 0.53?

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch!

**Comment by stuartarchibald** [comment]
> Will this need to be cherry-picked onto the release branch for 0.53?

Think so.

**Pull request merged**